packages/xo-web/src/common/xo/index.js

```diff
@@ -1381,8 +1381,20 @@ export const hidePcis = async (pcis, hide) => {
       title: _(hide ? 'pcisEnable' : 'pcisDisable', { nPcis: pcis.length }),
     })
   } catch (error) {
-    return
+    if (noHostsAvailableErrCheck(error)) {
+      try {
+        await confirm({
+          body: _('hostEvacuationFailed'),
+          title: _('confirmForceRebootHost'),
+        })
+
+        return _call('pci.disableDom0Access', { pcis: resolveIds(pcis), disable: hide, force: true })
+      } catch (error) {
+        return
+      }
+    }
```

Comment:
You're not going to catch the call error here since the call happens after the test. This is just the `catch` of `confirm`. So it needs to look like this:
```js
await confirm(...)

try {
	call(..., { ... })
} catch (err) {
	if (!noHostsAvailable.is(err) {
		throw err
	}

	await confirm(...)
	await call(..., { ..., force: true })
}
```

It shouldn't be necessary to catch the `confirm` modal's errors since there should be only one possible error which is `undefined` when the modal is closed without clicking on "OK". And an `undefined` error should be properly handled already by `ActionButton` or anything that called this method. But maybe check that it's indeed the case.